backend/management/commands/weekly_summary_campaign.py

```diff
@@ -27,6 +28,11 @@ def format_email_content(shared_items, shared_subscriptions):
                 f"- {subscription.name} (shared by {subscription.owner.username})<br>"
             )
 
+    if shared_requests:
+        content += "<h2>📢 Requests shared with you:</h2>"
```

Comment:
We should consider changing the text here. maybe instead say, "Items or subscriptions the community is seeking out" ?